alarm/weston-rpi/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# $Id$
+# Maintainer: S?bastien Luttringer
+# Contributor: Joel Teichroeb <joel@teichroeb.net>
+
+# ALARM: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
+#  - export flags for rpi
+#  - build for both armv6h and armv7h
+
+buildarch=20
+
+pkgname=weston-rpi
+pkgver=1.7.0
+pkgrel=1
+pkgdesc='Reference implementation of a Wayland compositor'
+arch=('armv6h' 'armv7h')
+url='http://wayland.freedesktop.org'
+license=('MIT')
+depends=('libxkbcommon' 'libinput' 'libunwind' 'poppler-glib' 'mtdev' 'libxcursor' 'glu' 'pango' 'colord')
+makedepends=('raspberrypi-firmware')
+replaces=('weston')
```

Comment:
And it will replace any installed weston package on any other device with an rpi-specific one.  Please review our list of supported devices to confirm for yourself that there are, in fact, other devices aside from the pi, and we support many of them.
http://archlinuxarm.org/platforms